instrumentation/spring/spring-webmvc-3.1/javaagent/src/main/java/org/springframework/web/servlet/OpenTelemetryHandlerMappingFilter.java

```diff
@@ -28,18 +31,41 @@
 import org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping;
 
 public class OpenTelemetryHandlerMappingFilter implements Filter, Ordered {
+  // copied from ServletRequestPathUtils
+  private static final String PATH_ATTRIBUTE =
+      "org.springframework.web.util.ServletRequestPathUtils.PATH";
```

Comment:
I happened to click into an earlier version of `ServletRequestPathUtils` and noticed they used lowercase `.path` there, but uppercase `.PATH` in later versions.

maybe we can get the constant via reflection instead?